<?xml version="1.0"?>
<launch>


  <include file="$(find setup)/launch/p3at_spawn.launch">
    <arg name="robotname" value="robot1"/>
    <arg name="initialy" value="5"/>
    <arg name="initialx" value="5"/>
  </include>




  <include file="$(find setup)/launch/p3at_spawn.launch">
    <arg name="robotname" value="robot2"/>
    <arg name="initialy" value="-5"/>
    <arg name="initialx" value="-5"/>
  </include>




</launch>
